pyfr/__main__.py

```diff
@@ -53,8 +56,10 @@ def main():
                               help='input solution files')
     ap_partition.add_argument('outd', help='output directory')
     partitioners = sorted(cls.name for cls in subclasses(BasePartitioner))
-    ap_partition.add_argument('-p', dest='partitioner', choices=partitioners,
+    ap_partition.add_argument('-l', dest='partitioner', choices=partitioners,
```

Comment:
It might be worth adding examples of the `-l` and `-p` flags to the running pyfr examples in the docs. Or to the CLI docs that we now have.

Reply:
Including `-p` I think is a good call.  `-l` probably just for the docs as otherwise it will confuse our users (as the default tries all available libraries). @pv101 Are you okay to action?